apiServer.js

```diff
@@ -83,11 +83,15 @@ app.get('/libraries', function(req, res) {
 
   res.setHeader("Expires", new Date(Date.now() + 360 * 60 * 1000).toUTCString());
   var fields = (req.query.fields && req.query.fields.split(',')) || [];
+  
   if (req.query.search) {
+    
     var searchParams = {
       typoTolerance: 'min', // only keep the minimum typos
-      hitsPerPage: 1000 // maximum
+      page: req.query.page || 0, // default
+      hitsPerPage: req.query.hitsPerPage || 1000 // maximum
```

Comment:
I don't think that the `hitsPerPage` default should be 1000, a more sane value would be 20, which is the default

Reply:
By all means, I agree. Just keeping the PR to the minimum amt of changes (how many people use the api now expecting hundreds of results, now that its been that way for years)

Reply:
Ah, I misunderstood, I thought this part was the website, but it's the API, good for me 💯

Reply:
We can change this later :)